static/js/services/report-view-model-generator.js

```diff
@@ -32,11 +32,16 @@ angular.module('inboxServices').factory('ReportViewModelGenerator',
             label: label,
             depth: depth
           });
-          getFields(results, value, label, depth + 1);
+          getFields(doc, results, value, label, depth + 1);
         } else {
+          var filePath = ['user-file']
+                  .concat(labelPrefix.split('.').slice(1))
+                  .concat([key])
+              .join('/');
           results.push({
             label: label,
             value: value,
+            file: doc._attachments[filePath],
```

Comment:
Makes sense.  I guess we can do similar to the `jr://` images inside forms - show a spinner, and insert the image with jQuery after the page has been rendered